modules/ROOT/pages/test_data.adoc

```diff
@@ -0,0 +1,7 @@
+= Test data
+
+To be able to reach even better predictions, we need to train our algorithm on real-world data.
+
+These data can help us try out and test new versions of our algorithm. It means also that in later version of the modules there will be higher chances that the prediction made but the module fit your own processes.
+
+If you are willing to help us in that way, follow the procedure indicated in https://community.bonitasoft.com/blog/data-science-applied-bpm[this blog post], and sent it to link:mailto:rd.user@bonitasoft.com[rd.user@bonitasoft.com]
```

Comment:
⚠️ as we change the whole content, we should change this email address which is not monitored

Reply:
Ah... indeed. Will replace by product management.